### DISCUSSION TITLE
Incorrect syntax reported by OPA (rego_parse_error: illegal token)

### DISCUSSION THREAD

**Discussion by wedla** [open-policy-agent/community#337] [OPA and Rego]
Hello, guys, how are you doing? I hope everyone is safe and sound.

I have a question about the use of **glob.match** function. In the project I work, we are using OPA by downloading bundles, so we can evaluate if some credentials have permission or not to consume our resources. But we have an address that belongs to a websocket, and it should be in the safe endpoints.

The rego expression is the following:

```
allow {
    glob.match("/realtime/ws*", [], path)
}
```

But when OPA downloads the bundle, it throws the following error:

```
"level":"error","msg":"Bundle load failed: 1 error occurred: api_authz_token.rego:61: rego_parse_error: illegal token\n\t    glob.match(\"/realtime/ws*\", [], path)\n\t^","name":"authz","plugin":"bundle","time":"2023-02-07T22:21:52Z"
```

I read and re-read the documentation, but I wasn't able to find an answer for this error. On the rego playground, I've received a positive response by evaluating the expression above, as it follows: https://play.openpolicyagent.org/p/onjH19HICw

So can someone help me?

**Comment by srenatus** [comment]
Can you try removing all whitespace before glob.match? Just to be sure there's no character there that looks like a blank but isn't.
